TelePharo-Core.package/TlpRemoteUIManager.class/instance/registerRemoteTool.as..st

```diff
@@ -4,7 +4,13 @@ registerRemoteTool: aSeamlessProxy as: aNameSelector
 	| existingTool |
 	self shouldBeRegistered.
 	existingTool := Smalltalk tools remoteToolNamed: aNameSelector.
-	existingTool isSeamlessProxy ifFalse: [ "otherwise new remote tools are connected"
-		localTools at: aNameSelector ifAbsentPut: existingTool].
+	
+	existingTool isSeamlessProxy 
+		ifFalse: [ "otherwise new remote tools are connected" 
+			localTools at: aNameSelector ifAbsentPut: existingTool].
 		
-	Smalltalk tools registerRemoteTool: aSeamlessProxy as: aNameSelector
\ No newline at end of file
+	Smalltalk tools registerRemoteTool: aSeamlessProxy as: aNameSelector.
+	
+	(Smalltalk tools remoteToolNamed: aNameSelector) isSeamlessProxy
```

Comment:
It will be always true because we just registered a proxy. So just do the cleanup